pkg/build/util/util_test.go

```diff
@@ -84,3 +90,219 @@ func TestTrustedMergeEnvWithoutDuplicates(t *testing.T) {
 	}
 
 }
+
+var credsRegex = regexp.MustCompile(".*user:password.*")
+var redactedRegex = regexp.MustCompile(".*redacted*")
```

Comment:
missing '.' - should be (".\*redacted.\*"), or better strings.Contains(s, "redacted") throughout - leading and trailing .\* are superfluous.